pyfar/classes/audio.py

```diff
@@ -95,11 +95,40 @@ def reshape(self, newshape):
                 newshape + (length_last_dimension, ))
         except ValueError:
             if np.prod(newshape) != np.prod(self.cshape):
-                raise ValueError((f"Can not reshape audio object of cshape "
+                raise ValueError((f"Cannot reshape audio object of cshape "
                                   f"{self.cshape} to {newshape}"))
 
         return reshaped
 
+    def transpose(self, *axes):
+        """Transpose time/frequency data and return copy of the audio object.
+
+        Parameters
+        ----------
+        axes : empty, `None`, iterable of ints, or n ints
+            empty (default) or `None`: reverses the order of `self.caxes`.
+            iterable of ints: `i` in the `j`-th place of the interable means
+            that the `i`-th axis becomes transposed object's `j`-th axis.
+            n ints: same as 'iterable of ints'.
+        """
+        if hasattr(axes, '__iter__'):
+            axes = axes[0] if len(axes) == 1 else axes
+        if axes is None or len(axes) == 0:
+            axes = tuple(range(len(self.cshape)))[::-1]
+        else:
+            tuple([a-1 if a < 0 else a for a in axes])
```

Comment:
I think it is fine, this will lead to errors in both 
```python
np.empty(np.ones(len(self.cshape), dtype=int)).transpose(axes)
``` 
as well as
```python
transposed._data = transposed._data.transpose(*axes, len(self.cshape))
```